x-pack/plugins/observability/public/config/register_alerts_table_configuration.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AlertsTableConfigurationRegistryContract } from '@kbn/triggers-actions-ui-plugin/public';
+import { Storage } from '@kbn/kibana-utils-plugin/public';
+
+import { TGridModel } from '@kbn/timelines-plugin/public';
+import { observabilityFeatureId } from '../../common';
+import { ALERT_TABLE_STATE_STORAGE_KEY } from '../pages/alerts/containers/alerts_page/alerts_page';
```

Comment:
@academo Please note that recently we parameterized the alerts table storage key as part of this PR https://github.com/elastic/kibana/pull/130538, since we have 2 pages in observability that use the alerts table a) Alerts page and b) Overview page